packages/shared/components/molecules/TransactionDetails.svelte

```diff
@@ -123,36 +123,22 @@
         ...(claimedTime && { claimedTime: { data: claimedTime } }),
     }
 
-    let copyableDetailsList: { [key in string]: { data: string; tooltipText?: string } }
-    $: copyableDetailsList = {
-        ...(governorAddress && {
-            governorAddress: { data: governorAddress },
-        }),
-        ...(stateControllerAddress && {
-            stateControllerAddress: { data: stateControllerAddress },
-        }),
+    function handleTransactionIdClick(): void {
+        explorerUrl
+            ? Platform.openUrl(`${explorerUrl}/block/${claimingTransactionId}`)
+            : setClipboard(claimingTransactionId)
     }
 
     function getDateFormat(date: Date): string {
-        try {
-            if (date) {
-                return formatDate(date, {
-                    dateStyle: 'long',
-                    timeStyle: 'medium',
-                })
-            } else {
-                return undefined
-            }
-        } catch {
+        if (date) {
```

Comment:
you got rid of this try catch but this will throw an error crashing firefly if it fails? Are we sure it won't fail here?

Reply:
reverted that change